evasquez26/angular4-startup
# S4-ONE-212 School List

## Story / Task

see [ONE-212](https://bixlabs.atlassian.net/browse/ONE-212)


## What this PR does?
Adds functionality in order to display a list of organizations


### How to test / reproduce

- Login to the app
- Go to Schools menu
- You should see a list of organizations

### TODOs
- [x] Rebase after mergin #35 

### Related PRs
N/A

### Migrations

Please, run `npm install`


Thanks
